fhir-tools/pom.xml

```diff
@@ -295,29 +295,26 @@
             </properties>
         </profile>
         <profile>
-            <id>fhir-javadocs</id>
```

Comment:
can we stop deploying this artifact?  it doesn't seem real useful to have this one in maven central.
it probably makes sense to leave it out of the javadocs too.

Reply:
all done